editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -1687,13 +1687,18 @@ bool CanvasItemEditor::_gui_input_anchors(const Ref<InputEvent> &p_event) {
 			_commit_canvas_item_state(
 					drag_selection,
 					vformat(TTR("Move CanvasItem \"%s\" Anchor"), drag_selection.front()->get()->get_name()));
+			snap_target[0] = SNAP_TARGET_NONE;
+			snap_target[1] = SNAP_TARGET_NONE;
```

Comment:
This looks like something that should be a part of `_reset_drag()` method.

Reply:
> This looks like something that should be a part of `_reset_drag()` method.

If desired i can update my pr to do that

Reply:
It's better to do it in a separate PR.